### PR TITLE
Update Ruby protobuf to 3.0.0 (GA), and gRPC to 1.0.0.pre2

### PR DESCRIPTION
**Pull request by jmuk**

### CODE REVIEW

**Comment by jmuk** [comment]
part of https://github.com/googleapis/gax-ruby/issues/37

File: config/dependencies.yml [hunk 1]
```diff
@@ -49,15 +49,15 @@ gax:
     version: '0.12.5'
     next_version: '0.13.0'
   ruby:
-    version: '0.4.1'
+    version: '0.5.0'
```
Comment (geigerj) [inline config/dependencies.yml]: https://github.com/googleapis/gax-ruby/pull/38 bumps the version to `0.4.4`, not `0.5.0`. Is this in anticipation of more changes?

Comment (jmuk) [inline config/dependencies.yml]: Thanks for catching, that was my mistake. Fixed to 0.4.4.

**Comment by geigerj** [comment]
LGTM

**Pull request merged**